test/llvm/vectorization/SKIPIF

```diff
@@ -0,0 +1 @@
+CHPL_LLVM!=llvm
```

Comment:
Or something... never sure what the red characters in GitHub here at the end mean.